tests/integration/records/single_db/test_records_load.py

```diff
@@ -51,16 +52,30 @@ def test_load_error(self):
     def test_load_from_s3_records_directory(self):
         try:
             import boto3  # noqa
+            import s3_concat  # noqa
         except ModuleNotFoundError:
-            logger.warning("Not running in test environment with boto3, "
+            logger.warning("Not running in test environment with boto3/s3_concat, "
```

Comment:
Turns out that the BigQuery integration tests originally failed.  The reason:

* The BigQuery integration tests use the 'bigquery' extra in setup.py, which includes the `smart_open` package.
* The `smart_open` package has a transitive dependency on `boto3`, the AWS API client library.
* To actually import the `record_mover.url.s3.s3_file_url` module and thus be able to use `s3://` URLs, we need both `boto3` and `s3_concat` - but in this scenario, we only have one.

So let's test more completely here before trying to run a scenario which isn't always possible given the integration test environment.